draft-ietf-jsonpath-base.md

```diff
@@ -1313,6 +1310,9 @@ appear in the list.
 Note that any node matched by more than one selector is kept
 as many times in the nodelist.
 
+Where a selector can produce a nodelist in more than one possible order, the
+selector may produce a nodelist in a distinct order each time it appears in the child segment.
```

Comment:
Is this just about appearance?  Or evaluation?

Reply:
I thought evaluation was clear, but perhaps not. Clarified in de0b902dca07cd68106d73b19b84b6c45c2cf2c6.